backend/apps/owasp/management/commands/owasp_run_notification_worker.py

```
@@ -0,0 +1,113 @@
+            
+            if not users.exists():
+                logger.info("No active users found.")
+                return
+
+            logger.info(f"Sending snapshot notification to {users.count()} users")
+            
+            for user in users:
+                self.send_notification(user, snapshot)
+
+        except Snapshot.DoesNotExist:
+            logger.error(f"Snapshot matching ID {snapshot_id} not found.")
```

Ruff (0.14.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)